app/models/user.rb

```diff
@@ -21,9 +21,8 @@
 #
 class User < ApplicationRecord
   # Include default devise modules. Others available are:
-  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
-  devise :database_authenticatable, :registerable,
-         :recoverable, :rememberable, :validatable
+  # :confirmable, :lockable, :timeoutable, :trackable, :rememberable, :validatable and :omniauthable
+  devise :database_authenticatable, :registerable
```

Comment:
I think need to add :confirmable also

Reply:
@VitaliiShevchenko since emails will be entered by a manager via ActiveAdmin `:confirmable` isn't needed. In fact companies will get angry if their employees get sent confirmation emails like that when setting up a SaaS like this one.